### DISCUSSION TITLE
add tool with emojis in editors

### DISCUSSION THREAD

**Discussion by Lucasm4sco** [forem/forem#20275] [Feature Requests]
Most of the time I use dev.to, I'm on the PC, and as a user I really like to comment using emojis, making me have to look for emojis on the web whenever I want to use one, I don't think this only happens to me, I see several posts using emojis and maybe it also happens with them, so one feature I would like to see would be the ability to use emojis through the editing tool when writing articles or comments.

I would also like to know what you think about this.

**Comment by michael-tharrington** [comment]
Love this idea! Emojis would be a great addition to the toolbar.
